<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="INBNRMARITA" failures="1" tests="5" name="Admin- Close SO" time="45.671" errors="0" timestamp="2022-07-28T13:13:17 IST">
  <testcase classname="script.NewClosePoConsumer" name="poClose" time="7.216"/>
  <testcase classname="script.NewClosePoConsumer" name="resetButton" time="0.034"/>
  <testcase classname="script.NewClosePoConsumer" name="validateCropList" time="0.089"/>
  <testcase classname="script.NewClosePoConsumer" name="companyName" time="0.044"/>
  <testcase classname="script.NewClosePoConsumer" name="closePO" time="31.612">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at script.NewClosePoConsumer.closePO(NewClosePoConsumer.java:93)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- closePO -->
</testsuite> <!-- Admin- Close SO -->
